tests/cypress/tests/10-managed/01-verify-managed-cluster.spec.js

```diff
@@ -23,23 +23,19 @@ describe(
       searchPage.whenGoToSearchPage()
     })
 
+    it(`[P3][Sev3][${squad}] should validate the endpoint version for the managed clusters are accurate`, function () {
+      searchPage.shouldValidateManagedCluster()
+    })
+
     context(
       'verify: managed cluster resource endpoint',
       { tags: tags.modes },
       () => {
-        it(`[P3][Sev3][${squad}] should validate the endpoint version for the managed clusters are accurate`, function () {
-          searchPage.shouldValidateManagedCluster()
-        })
-
         it(`[P3][Sev3][${squad}] should verify endpoint pods are all in running state`, function () {
-          searchPage.shouldVerifyManagedClusterPodsAreRunning(this.clusterName)
-        })
-
-        it(`[P3][Sev3][${squad}] should verify the yaml information is correct and there are no errors in the logs`, function () {
-          searchPage.shouldVerifyManagedClusterPodsAreRunning(
-            this.clusterName,
-            true
-          )
```

Comment:
Removed this managed cluster test. Currently, checking the logs for errors could be problematic. There could be an error within the logs, which self resolves; however, this would cause our test case to fail. It would be better overall to remove this test case, plus we're also verifying managed cluster resource logs and YAML within earlier test suites.